lib/src/soloud.dart

```diff
@@ -145,6 +145,9 @@ interface class SoLoud {
   /// The backing private field for [isInitialized].
   bool _isInitialized = false;
 
+  /// Whether or not is it possible to ask for wave and FFT data.
+  bool isVisualizationEnabled = false;
```

Comment:
I would prefer this to be a getter, backed by a private member (`_isVisualizationEnabled`). Client code should not be able to set (change) this field.

Reply:
In fact, maybe this field should just be private. It seems weird to provide both `getVisualizationEnabled()` (an async method) and `isVisualizationEnabled` (a sync getter / field).